### ISSUE TITLE
Craete versioning scheme for zip and mp4 URLs

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/docs#1968]
...so that we can set these to immutable to always serve them from the CDN.

**Comment by patrickkettner** [comment]
@sebastianbenz I beleive thumbor takes care of this for images, right? If so, is the task scoped to versioning  videos?

**Comment by sebastianbenz** [comment]
Yes, it's working for thumbor URLs, but we need to set the corresponding
cache headers to make them immutable. It's less critical for videos etc.

>
